### PR TITLE
feat: added authentication

### PR DESCRIPTION
**Pull request by paarthbajaj**
signin
signup

### CODE REVIEW

File: signup.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        </label>
+        <label class="log-input">
+          <input
+            class="input log-input req-input"
+            type="password"
+            placeholder="Password"
+            pattern="^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$"
+            required
+          />
+          <span class="error"
```
Comment (Priya-Patel31) [inline signup.html:52]: refactor : avoid span tag as it is non-semantic tag

**Review by munjalSaksham71 — APPROVED** [review]
Looks Good to merge

File: signup.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        <label class="log-input">
+          <input
+            class="input log-input req-input"
+            type="email"
+            placeholder="Email"
+            required
+          />
+          <span class="error">Please enter your email</span>
+        </label>
+        <label class="log-input">
```
Comment (Priya-Patel31) [inline signup.html:44]: same as above

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    <title>Cart</title>
+  </head>
+  <body class="log-page">
+    <!-- *****************WEBSITE HEADER***************** -->
+    <header class="login-header pt-5">
+      <div class="logo-container brand-logo">Quirky Face</div>
+    </header>
+    <main class="log-main m-radius">
+      <h1 class="txt-center">Sign Up to BobbleHead</h1>
+      <form class="log-form flex-column g-1 align-center justify-center">
```
Comment (Priya-Patel31) [inline signup.html:25]: what is this g-1 class doing ??
You could have name something better.

**Review by Omkar-Chaskar — APPROVED** [review]
Good to merge

File: signup.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        <label class="log-input">
```
Comment (Priya-Patel31) [inline signup.html:26]: refactor : add for attribute as it good accessibility practice inside label tag

File: signup.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    <link rel="stylesheet" href="./homepage.css" />
+    <link rel="stylesheet" href="./product-listing.css" />
+    <link rel="stylesheet" href="./cart.css" />
+    <link rel="stylesheet" href="./wishlist.css" />
+    <link rel="stylesheet" href="./auth.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    />
+    <title>Cart</title>
```
Comment (Priya-Patel31) [inline signup.html:16]: fix : Change the title to signup page

**Review by Priya-Patel31 — APPROVED** [review]
Naming convention could be used better and there are some minor changes which need to be made rest everything looks good to me.

**Pull request merged**